Plugins/Public/playercntl_plugin/Givecash.cpp

```diff
@@ -413,19 +431,25 @@ namespace GiveCash
 		// of the transfer. The ini is cleared when ever the character logs in.
 		else
 		{
-			wstring msg = L"You have received " + ToMoneyStr(cash) + L" credits from " + ((bAnon) ? L"anonymous" : wscCharname);
 			LogTransfer(wscTargetCharname, msg);
 		}
 
-		AddLog("NOTICE: Send %s credits from %s (%s) to %s (%s)",
+		AddLog("NOTICE: Send %s credits from %s (%s) to %s (%s) %s %s",
 			wstos(ToMoneyStr(cash)).c_str(),
 			wstos(wscCharname).c_str(), wstos(HkGetAccountID(HkGetAccountByCharname(wscCharname))).c_str(),
-			wstos(wscTargetCharname).c_str(), wstos(HkGetAccountID(HkGetAccountByCharname(wscTargetCharname))).c_str());
+			wstos(wscTargetCharname).c_str(), wstos(HkGetAccountID(HkGetAccountByCharname(wscTargetCharname))).c_str(),
+			wscComment.size() ? "| With comment:" : "", wstos(wscComment).c_str());
 
 		// A friendly message explaining the transfer.
-		msg = L"You have sent " + ToMoneyStr(cash) + L" credits to " + wscTargetCharname;
+		msg = time + L" You have sent " + ToMoneyStr(cash) + L" credits to " + wscTargetCharname;
 		if (bAnon)
 			msg += L" anonymously";
+		if (wscComment.size())
+			msg += L" with comment";
```

Comment:
colon, space, wscComment?

Reply:
But why do we need to duplicate comment? Commands you typing are being saved to chat.

Reply:
okay, let's at least fix this though:
```suggestion
			msg += L" with a comment";
```